src/data/dist/building-statistics-by-property-type.json

```diff
@@ -0,0 +1 @@
: 426930.0, "max": 80514977.7, "twentyFifthPercentile": 4868874.475, "median": 6439100.15, "seventyFifthPercentile": 7406275.425}, "GrossFloorArea": {"mean": 86416.938, "count": 48, "min": 50111, "max": 256742, "twentyFifthPercentile": 63254.25, "median": 70526.0, "seventyFifthPercentile": 86906.75}, "SourceEUI": {"mean": 526.108, "count": 48, "min": 316.3, "max": 1618.9, "twentyFifthPercentile": 460.725, "median": 504.0, "seventyFifthPercentile": 547.275}, "SiteEUI": {"mean": 260.66, "count": 48, "min": 138.7, "max": 1332.5, "twentyFifthPercentile": 213.075, "median": 232.9, "seventyFifthPercentile": 263.925}}, "Wholesale Club/Supercenter": {"GHGIntensity": {"mean": 13.44, "count": 5, "min": 12.1, "max": 15.1, "twentyFifthPercentile": 12.8, "median": 13.3, "seventyFifthPercentile": 13.9}, "TotalGHGEmissions": {"mean": 1930.94, "count": 5, "min": 1413.2, "max": 2359.2, "twentyFifthPercentile": 1807.6, "median": 1970.5, "seventyFifthPercentile": 2104.2}, "ElectricityUse": {"mean": 10237486.52, "count": 5, "min": 7113289.6, "max": 12796546.3, "twentyFifthPercentile": 9650313.1, "median": 10519924.9, "seventyFifthPercentile": 11107358.7}, "NaturalGasUse": {"mean": 6273151.94, "count": 5, "min": 4462121.6, "max": 8705550.0, "twentyFifthPercentile": 5676649.2, "median": 5704614.0, "seventyFifthPercentile": 6816824.9}, "GrossFloorArea": {"mean": 142737.2, "count": 5, "min": 116720, "max": 155765, "twentyFifthPercentile": 141676.0, "median": 148364.0, "seventyFifthPercentile": 151161.0}, "SourceEUI": {"mean": 245.46, "count": 5, "min": 222.0, "max": 276.0, "twentyFifthPercentile": 232.8, "median": 241.2, "seventyFifthPercentile": 255.3}, "SiteEUI": {"mean": 115.2, "count": 5, "min": 104.9, "max": 127.2, "twentyFifthPercentile": 108.2, "median": 109.8, "seventyFifthPercentile": 125.9}}, "Worship Facility": {"GHGIntensity": {"mean": 24.6, "count": 2, "min": 6.3, "max": 42.9, "twentyFifthPercentile": 15.45, "median": 24.6, "seventyFifthPercentile": 33.75}, "TotalGHGEmissions": {"mean": 2645.85, "count": 2, "min": 1351.2, "max": 3940.5, "twentyFifthPercentile": 1998.525, "median": 2645.85, "seventyFifthPercentile": 3293.175}, "ElectricityUse": {"mean": 3713316.4, "count": 2, "min": 1302350.0, "max": 6124282.8, "twentyFifthPercentile": 2507833.2, "median": 3713316.4, "seventyFifthPercentile": 4918799.6}, "NaturalGasUse": {"mean": 38902872.2, "count": 2, "min": 7444705.9, "max": 70361038.5, "twentyFifthPercentile": 23173789.05, "median": 38902872.2, "seventyFifthPercentile": 54631955.35}, "GrossFloorArea": {"mean": 153900.0, "count": 2, "min": 91800, "max": 216000, "twentyFifthPercentile": 122850.0, "median": 153900.0, "seventyFifthPercentile": 184950.0}, "SourceEUI": {"mean": 480.05, "count": 2, "min": 115.6, "max": 844.5, "twentyFifthPercentile": 297.825, "median": 480.05, "seventyFifthPercentile": 662.275}, "SiteEUI": {"mean": 421.7, "count": 2, "min": 62.8, "max": 780.6, "twentyFifthPercentile": 242.25, "median": 421.7, "seventyFifthPercentile": 601.15}}}
```

Comment:
This will be some interesting data to play with and render later - I wonder if we should add a list of the output data files to the `README`, especially since there's a lot here we don't use yet